packages/next/shared/lib/constants.ts

```diff
@@ -83,16 +83,4 @@ export const EDGE_UNSUPPORTED_NODE_APIS = [
   'CountQueuingStrategy',
   'DecompressionStream',
   'DomException',
-  'Event',
-  'EventTarget',
-  'MessageChannel',
-  'MessageEvent',
-  'MessagePort',
-  'ReadableByteStreamController',
-  'ReadableStreamBYOBRequest',
-  'ReadableStreamDefaultController',
-  'TextDecoderStream',
-  'TextEncoderStream',
-  'TransformStreamDefaultController',
-  'WritableStreamDefaultController',
```

Comment:
**question:** so... does the edge runtime supports all of them? What a good news!